tests/integration/test_cms_api.py

```diff
@@ -155,9 +155,14 @@ def test_get_datasets_suggestions_returns_correct_suggestions(
     resp = client.get('/cms/datasets?query=datas',
                       headers=auth_headers_for_user(users['cms_user']))
 
+    assert resp.json == []
+
+    resp = client.get('/cms/datasets?query=datas*',
+                      headers=auth_headers_for_user(users['cms_user']))
+
     assert resp.json == ['dataset1', 'dataset2']
 
-    resp = client.get('/cms/datasets?query=another_',
+    resp = client.get('/cms/datasets?query=*other_*',
```

Comment:
wildcard at the end should be always added automatically (because we want to have search-as-type functionality)